pjmedia/src/pjmedia/transport_srtp_dtls.c

```diff
@@ -1361,6 +1374,50 @@ static pj_status_t dtls_on_recv(pjmedia_transport *tp, unsigned idx,
         (ds->setup == DTLS_SETUP_ACTPASS || ds->setup == DTLS_SETUP_PASSIVE))
     {
         pj_status_t status;
+        pj_bool_t check_hello_addr = PJ_FALSE;
```

Comment:
Is this still needed if the check is already done in ICE?

And if we don't use ICE, should we compare it against the SDP?
I don't mind if we don't, but we need to document it somewhere